test/Api.Tests.Nunit/ControllerTests/UsersControllerTest.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Api.Controllers;
+using Api.Models;
+using Api.Models.Entity;
+using MyTested.AspNetCore.Mvc;
+using NUnit.Framework;
+
+namespace Api.Tests.Nunit.ControllerTests
+{
+    [TestFixture]
+    public class UsersControllerTest
+    {
+        /// <summary>
+        /// Tests response from Get action of UsersController.
+        /// </summary>
+        [Test]
+        public void GetUserListTest()
+        {
+            MyMvc
+                .Controller<UsersController>()
+                .Calling(c => c.Get(null, null, 1))
+                .ShouldReturn()
+                .Ok();
+        }
+
+        /// <summary>
+        /// Tests response code 403 from Put(Id,AdminUserFormModel) action of UsersController
+        /// </summary>
+        [Test]
+        public void UpdateUserTestResponseCode403()
+        {
+            MyMvc
+                .Controller<UsersController>()
+                .WithAuthenticatedUser(user => user.WithClaim("Id", "1")) // add claim with custom name and value of 1                                
+                .Calling(c => c.Put(4, new AdminUserFormModel
+                {
+                    FirstName = "First Name",
+                    LastName = "Last Name",
+                    Role = "InvalidRole"
+                }))                
+                .ShouldReturn()
+                .StatusCode(403);
+        }
+
+        /// <summary>
+        /// Tests response from Put(Id,AdminUserFormModel) action of UsersController
+        /// </summary>
+        [Test]
+        public void UpdateUserTest()
+        {
+            MyMvc
+                .Controller<UsersController>()
+                .WithAuthenticatedUser(user => user.WithClaim("Id", "1")) // add claim with custom name and value of 1                                                
+                .WithDbContext(dbContext =>
```

Comment:
".WithDbContext(dbContext =>" here you are testing only the MVC Stack. But desirable will be to test also all EF core related thing. So you should try at least to use a InMemoryDatabase.